src/Jira/Api.php

```diff
@@ -453,6 +454,7 @@ public function search($jql, $startAt = 0, $maxResult = 20, $fields = '*navigabl
                 'startAt' => $startAt,
                 'maxResults' => $maxResult,
                 'fields' => $fields,
+                'expand' => $expand,
```

Comment:
Better not to rely on fact, that `http_build_query` is used in used in underlying client class and do the following:

1. move out the parameter array into a separate variable
2. add an IF statement that would check for `isset($expand)` and only then add array entry
3. use that array in `->api` call